frontend/src/components/nodes/DatamapsExtension.tsx

```diff
@@ -0,0 +1,355 @@
+import Datamaps from 'datamaps';
+import * as d3 from 'd3';
+
+import React from 'react';
+
+interface Props {
+  arc?: any[];
```

Comment:
what is this prop? maybe give it full name to better understand

Reply:
That prop is for circle arcs that can be drawed between 2 points on map, anyway it was an unused prop and i just cleaned the extension of unused props